ostd/src/arch/loongarch/trap/trap.rs

```diff
@@ -10,11 +10,10 @@ global_asm!(include_str!("trap.S"));
 ///
 /// # Safety
 ///
-/// This function will:
-/// - Set `eentry` to `trap_entry`
```

Comment:
It seems that this comment is kept in RISC-V version. We previously also kept the x86's comment. Then I wander why remove it in the LoongArch version?
https://github.com/asterinas/asterinas/blob/121da78aa2576da02294662ee1ca279aa1316c0f/ostd/src/arch/x86/trap/mod.rs#L105-L114

Maybe it's incomplete?
```rust
/// This function will:
/// - Set `ecfg`'s VS bit to zero.
/// - Set `eentry` to the trap entry.
```